### ISSUE TITLE
[Feature][task-datax] Added Doris visualization configuration in datax

### ISSUE THREAD

**Issue by xinxingi** [apache/dolphinscheduler#14615]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description
I found the corresponding code path to support Doris org.apache.dolphinscheduler.plugin.task.datax.DataxTask, the method to generate JSON configuration file is buildDataxJobContentJson(). I reviewed the method and found writer.parameter.* or read.parameter.* many parameters are missing. Therefore, an enhancement should be made to the corresponding code of datax to adapt to more data source plug-in access.

![image](https://github.com/apache/dolphinscheduler/assets/49302071/222ea116-d08f-4ff0-aa58-ccbfecb07d80)

Since I am better at data development in my work, I hope that more capable contributors can receive this task. If not, I'll finish developing the feature.
### Use case

Just like configuring the mysql plug-in, you can complete the configuration of doris on the page.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by xinxingi** [comment]
I found the corresponding code path to support Doris
org.apache.dolphinscheduler.plugin.task.datax.DataxTask, the method to generate JSON configuration file is buildDataxJobContentJson(). I reviewed the method and found writer.parameter.* or read.parameter.* many parameters are missing. Therefore, an enhancement should be made to the corresponding code of datax to adapt to more data source plug-in access.
